### PR TITLE
Upgrade mender-client-acceptance-testing to Ubuntu 20.04

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio@tranchitella.eu>

### CODE REVIEW

File: mender-client-acceptance-testing/Dockerfile [hunk 1]
```diff
@@ -11,7 +11,8 @@ RUN apt-get update -qq && apt-get install -yyq python3-pip && pip3 install --upg
 # Get OS requirements from master
 RUN apt-get install -yyq wget && \
     wget https://raw.githubusercontent.com/mendersoftware/meta-mender/master/tests/acceptance/requirements_deb.txt && \
-    apt-get install -yyq $(cat requirements_deb.txt) && apt-get install -yq linux-modules-`uname -r`
+    apt-get install -yyq $(cat requirements_deb.txt | sed -e 's/libvirt-bin/libvirt-clients/') && \
+    apt-get install -yq linux-modules-`uname -r`
```
Comment (kacf) [inline mender-client-acceptance-testing/Dockerfile:15]: Is `requirements_deb.txt` used anywhere else? Shouldn't we fix the original instead?

**Comment by lluiscampos** [comment]
I have to think about this one too. How to drop `warrior` support from here but still have a way to run tests if ever required :thinking:

**Comment by lluiscampos** [comment]
I expect this to have quite a few more problems in the client acceptance tests and needs thorough testing.

Furthermore I suspect that the real motivation was the update of the Raspbian testing image, while updating the yocto testing image is just a "once I was here...". The best would be to have two separate PRs as they impact different parts of the ecosystem.

File: mender-client-acceptance-testing/Dockerfile (hunk 1 above)
Comment (lluiscampos) [reply mender-client-acceptance-testing/Dockerfile:15]: Yes, updating the base image we can directly edit the `requirements_deb.txt` file. I suspect we might need new packages too: I have some local tweaks to make it work with Ubuntu 20.10

**Review by lluiscampos — CHANGES_REQUESTED** [review]
I have no objection in upgrading the raspbian base image for testing, but please let's move to a separate PR the yocto testing changes (which need to be tested carefully).

The raspbian base image update needs to be tested too, of course, but that it much simpler to do.

**Comment by tranchitella** [comment]
I think we can close this PR, and postpone the discussion about upgrading to Ubuntu 20.04 to a later point of time.
What was really important is the upgrade of the Raspian image to support ECC keys.

**Pull request closed without merging**